src/main/kotlin/com/ampnet/reportserviceeth/service/data/TransactionFactory.kt

```diff
@@ -44,22 +47,22 @@ abstract class Transaction(transaction: TransactionInfo) {
     var percentageInProject: String? = null
     var locale: Locale = Locale.ENGLISH
 
-    fun setPercentageInProject(expectedProjectFunding: BigInteger) {
-        if (expectedProjectFunding.signum() == 1) {
-            percentageInProject = amount
-                .multiply(BigInteger.valueOf(TO_PERCENTAGE))
-                .divide(expectedProjectFunding)
-                .toString().take(LENGTH_OF_PERCENTAGE)
-        }
-    }
+//    fun setPercentageInProject(expectedProjectFunding: BigInteger) {
+//        if (expectedProjectFunding.signum() == 1) {
+//            percentageInProject = amount
+//                .multiply(BigInteger.valueOf(TO_PERCENTAGE))
+//                .divide(expectedProjectFunding)
+//                .toString().take(LENGTH_OF_PERCENTAGE)
+//        }
+//    }
 
-    fun amountToCalculate(): BigInteger {
-        return when (txStatus) {
-            TransactionStatusType.PAID_IN -> amount
-            TransactionStatusType.PAID_OUT -> -amount
-            TransactionStatusType.UNDEFINED -> BigInteger.ZERO
-        }
-    }
+//    fun amountToCalculate(): BigInteger {
+//        return when (txStatus) {
+//            TransactionStatusType.PAID_IN -> amount
+//            TransactionStatusType.PAID_OUT -> -amount
+//            TransactionStatusType.UNDEFINED -> BigInteger.ZERO
+//        }
+//    }
```

Comment:
Check the logic for changed type in/out transactions or set: `TODO("Not implemented")`